django_project/config/defaults.py

```diff
@@ -19,3 +19,27 @@
     default=("FRONT_PAGE_RECENT_POST",),
     append=True,
 )
+
+register_setting(
+    name="CURRENT_VAT",
+    label="Current VAT",
+    description="The current VAT.",
+    editable=True,
+    default=15,
+)
+
+register_setting(
```

Comment:
hi @boney-bun 
is this OLD_VAT filled everytime we change CURRENT_VAT?

Reply:
thanks for the comments @meomancer 

a user should manually change the values of `OLD_VAT`, `CURRENT_VAT` and `CURRENT_VAT_STARTED`. 
I think the VAT will not frequently change so probably it's better to keep it simple.
if we need to keep the history of VAT changes, then I may need to modify this PR.